quinn-udp/Cargo.toml

```diff
@@ -14,14 +14,14 @@ workspace = ".."
 all-features = true
 
 [features]
-default = ["log"]
-# Write logs via the `log` crate when no `tracing` subscriber exists
-log = ["tracing/log"]
+default = ["tracing-log"]
+tracing-log = ["tracing", "tracing/log"]
 
 [dependencies]
 libc = "0.2.113"
+log = { workspace = true }
```

Comment:
Oh, forgot that this should be `optional = true`.

Reply:
@djc what should happen if the user enables neither the `tracing`, `tracing-log` nor `log` feature? Should there simply be no logging at all?

Fine by me. Though to me not intuitive. Given that `log` is so ubiquitous, I don't think anyone minds the non-optional dependency.

Reply:
tracing is mostly an alternative to logging, and most modern applications that I have seen get rid of log in favor of tracing. So yes, if both log and tracing are disabled, downstream crate will get no logging at all, which seems right.

Reply:
Reasoning makes sense. I updated https://github.com/quinn-rs/quinn/pull/1921/commits/91a667c3c00a1e63224c2dea0f0dfb9f7ba6f32f where both `tracing` and `log` are optional. `tracing` is the default. `tracing` takes precedence over `log`. Let me know if this looks good to you @djc. Thank you for bearing with me.